roseau/load_flow/models/tests/test_flexible_parameters.py

```diff
@@ -254,63 +262,3 @@ def test_flexible_parameter():
         fp.q_min = Q_(2.5e3, "kVAr")
     assert e.value.msg == "'q_min' must be greater than q_max (2.0 MVAr) but 2.5 MVAr was provided."
     assert e.value.code == RoseauLoadFlowExceptionCode.BAD_FLEXIBLE_PARAMETER_VALUE
-
-
-@pytest.fixture(params=["constant", "p_max_u_production"])
-def control_p(request) -> Control:
-    if request.param == "constant":
-        return Control.constant()
-    elif request.param == "p_max_u_production":
-        return Control.p_max_u_production(u_up=Q_(240, "V"), u_max=Q_(250, "V"))
-    elif request.param == "p_max_u_consumption":
-        return Control.p_max_u_production(u_up=Q_(210, "V"), u_max=Q_(220, "V"))
-    raise NotImplementedError(request.param)
-
-
-@pytest.fixture(params=["constant", "q_u"])
-def control_q(request) -> Control:
-    if request.param == "constant":
-        return Control.constant()
-    elif request.param == "q_u":
-        return Control.q_u(u_min=Q_(210, "V"), u_down=Q_(220, "V"), u_up=Q_(240, "V"), u_max=Q_(250, "V"))
-    raise NotImplementedError(request.param)
-
-
-@pytest.fixture(params=["keep_p", "euclidean"])
-def projection(request) -> Projection:
-    return Projection(type=request.param)
-
-
-@pytest.fixture()
-def flexible_parameter(control_p, control_q, projection) -> FlexibleParameter:
-    return FlexibleParameter(control_p=control_p, control_q=control_q, projection=projection, s_max=Q_(5, "kVA"))
-
-
-def test_plot(flexible_parameter):
```

Comment:
Do we still have this test somewhere?

Reply:
I've moved it to the engine part, as it was using the results